src/MavenUtils.ts

```diff
@@ -75,6 +115,17 @@ export default class MavenUtils {
     return false
   }
 
+  public static isSigningRequired(repoUrl: string): string | void {
+    if (this.isLocalMavenRepo(repoUrl)) {
+      return ``
```

Comment:
Please use `''` instead of backticks for this empty string (no interpolation needed).